src/metabase/server.clj

```diff
@@ -95,9 +95,12 @@
   ;;
   ;; TODO - I suppose the default value should be moved to the `metabase.config` namespace?
   (let [timeout (or (config/config-int :mb-jetty-async-response-timeout)
-                    (* 10 60 1000))]
+                    (* 10 60 1000))
+        handler (async-proxy-handler handler timeout)
+        stats-handler (doto (StatisticsHandler.)
```

Comment:
do we always want to initiate `statistichandler`?

I think we only want this if Prometheus is enabled right?

Reply:
We don't always want it. But I think its harmless enough that its better to not bring the notion of prometheus into the server/handler namespaces. If its not used it is harmless and there's way less branching in that namespace.